### ISSUE TITLE
Finetuning on another dataset

### ISSUE THREAD

**Issue by nishantb06** [pavancm/CONTRIQUE#12]
Hi 
Found the paper very useful and trying to Finetune on my own dataset. I wanted to know what the loss values looked like for your dataset. According to the experiments I am running, the loss doesn't converge and is stuck around 3.9-4 even after extensive training. 
- my dataset contains around 2.5 L image (100k synthetic + 150k UGC).
- Using a learning rate of 0.001 for later epochs.
Any Ideas on how to improve the training regimen so that the model converges?

**Comment by pavancm** [comment]
I need more details on what exactly you are experimenting. Are you trying to fine tune the model using ground truth quality scores? Or your data is different and using contrastive loss for training? Actual loss value is dependent on batch size in case of contrastive loss. Maybe drop me an email with a detailed description on the things you are trying to do.
